.gitignore

```diff
@@ -1,6 +1,7 @@
 # Generated by Cargo
 # will have compiled files and executables
 /target/
+/dotenv-lookup/target/
```

Comment:
I guess it's not needed when you work with workspace, because it shares target folder. Is not?